.gemspec

```diff
@@ -38,6 +38,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency(%q<locale>, "~> 2.1")
   s.add_runtime_dependency(%q<multi_json>, "~> 1.13")
   s.add_runtime_dependency(%q<httpclient>, "~> 2.8")
+  s.add_runtime_dependency(%q<concurrent-ruby>, "~> 1.0")
```

Comment:
Will need to add to `ext/project_data.yaml` too in the `gem_runtime_dependencies`. The `.gemspec` is only used when running from source/bundler, while project_data.yaml is used when building puppet as a gem. Puppetserver will also need the gem https://github.com/puppetlabs/puppetserver/blob/master/resources/ext/build-scripts/jruby-gem-list.txt.